website/main.py

```diff
@@ -16,66 +20,75 @@
     "yì qiū zhī hé",
     "yí mù liǎo rán",
 ]
-_characters_selector = "#table td.char"
 
-def set_table(characters):
-    for i, cell in enumerate(document.select(_characters_selector)):
-        cell.text = characters[i]
+# initial default values (a Java convention)
```

Comment:
We are writing Python code here, and Python generally does not have that convention.

But my point is, "because another language do X" may or may not be the reason we follow it. Let's discuss more on where that "define some default values beforehand" convention came from, whether that helps our code here, and whether we really want to change our standalone functions into this PR's global variables approach (global variables VS object-oriented approach is a fascinating topic in its own right).